src/Tag/Base/NormalTag.php

```diff
@@ -31,7 +31,12 @@ final protected function renderTag(): string
      */
     final public function open(): string
     {
-        return '<' . $this->getName() . $this->renderAttributes() . '>';
+        return '<' . $this->getName() . $this->renderAttributes() . '>' . $this->prepend();
```

Comment:
Looks like append, not prepend.

Reply:
my logic:
- before
- open tag 
- prepend 
- content of tag 
- append
- close tag
- after

Reply:
Ah. OK.